### PR TITLE
pfSense-pkg-cellular - add support for Quectel EC25 and symbolic link for NMEA port

### PR DESCRIPTION
**Pull request by kkondyl**
pfSense-pkg-cellular is updated to support Quectel EC25 and create a symbolic link for NMEA (GPS) port for Quectel EC25 and Simcom modems.

### CODE REVIEW

**Comment by kkondyl** [comment]
https://redmine.pfsense.org/issues/13063

File: net/pfSense-pkg-cellular/files/etc/devd/cellular.conf [hunk 1]
```diff
@@ -45,3 +51,18 @@ notify 1000 {
         action "%%PYTHON_CMD%% /usr/local/bin/cellular_dev.py -r";
 };
 
+# Quectel - EC25 LTE modem
+# ********
+attach 1000 {
+        match "vendor"          "0x2c7c";
+        match "product"         "0x0125";
+        action "/usr/local/bin/python3.8 /usr/local/bin/cellular_dev.py -a -d $device-name -m 2c7c0125";
```
Comment (jim-p) [inline net/pfSense-pkg-cellular/files/etc/devd/cellular.conf:59]: This should be using `%%PYTHON_CMD%%` and not the path to python directly.

File: net/pfSense-pkg-cellular/files/etc/devd/cellular.conf [hunk 2]
```diff
@@ -45,3 +51,18 @@ notify 1000 {
...
+};
+
+notify 1000 {
+        match "vendor"          "0x2c7c";
+        match "product"         "0x0125";
+        match "type"            "DETACH";
+        match "port"            "[0-9]";
+        action "/usr/local/bin/python3.8 /usr/local/bin/cellular_dev.py -r";
```
Comment (jim-p) [inline net/pfSense-pkg-cellular/files/etc/devd/cellular.conf:67]: This should be using `%%PYTHON_CMD%%` and not the path to python directly.

**Review by jim-p — CHANGES_REQUESTED** [review]

**Comment by jim-p** [comment]
As a part of this pull request, you must increase PORTVERSION or add a PORTREVISION line in the package Makefile. Without this version increase, the package will not be rebuilt to include the new change.

See https://docs.netgate.com/pfsense/en/latest/references/developer-style-guide.html#ports-packages-specific-rules

File: net/pfSense-pkg-cellular/files/etc/devd/cellular.conf (hunk 1 above)
Comment (kkondyl) [reply net/pfSense-pkg-cellular/files/etc/devd/cellular.conf]: thanks for the comment. Updated

File: net/pfSense-pkg-cellular/files/etc/devd/cellular.conf (hunk 2 above)
Comment (kkondyl) [reply net/pfSense-pkg-cellular/files/etc/devd/cellular.conf]: thanks for the comment. Updated

**Review by jim-p — APPROVED** [review]

**Pull request merged**